products/impulse/README.md

```diff
@@ -0,0 +1,88 @@
+|-----------|----------------------------------------------------------|--------------------------------------------------|
+| 1,2 - 1,3 | sedentary or lying lifestyle                             | Immobile, bedridden people                       |
+| 1,4 - 1,5 | sedentary work, little strenuous leisure time            | Office worker                                    |
+| 1,6 - 1,7 | Sitting, short-term walking and standing work            | student                                          |
+| 1,8 - 1,9 | Mostly walking and standing work                         | Salesperson, waiter, craftsman                   |
+| 2,0 - 2,4 | Physically demanding job or very active leisure activity | Construction worker, farmer, competitive athlete |
+
+#### Mental health
+
+> Disclaimer: This app will not replace a professional psychologist.
+
+The app should evaluate the mental condition of a user. This is important so that the app isn’t overwhelming the user with the baseline tasks. 
+
+### Baseline Tasks (Daily's)
+
+The baseline tasks are tasks that should be done every day to improve the lifestyle of a user. These tasks should be derived from the information which is given be the questionnaire. So that only relevant task that would improve the lifestyle would be chosen.
+
+Every baseline task should have information why it was selected for a user. There should also be information displayed on why this task is imported, so that there is a possible learning from the app.
+
+### Challenge Tasks
+
+Challenge Tasks are Tasks that are used motivate users to keep going and bring some variety to the app. These Challenges could include nutrition or fitness tasks. These challenges should also fit the lifestyle profile of a user.
+
+### Teams
+
+There should be the possibility to create teams for individual users. A user should also be part of implicit teams that are based on the location. Examples are city-based teams. 
+
+#### Company
+
+Companies should have the ability to create company teams for their business health management. It is import that now information of a user is visible to the company itself. These teams should be used for extra challenges or activities that are company related.
+
+Another extension to this is management of company organized sports courses. This for example includes the displaying of all necessary information for the course or the registration of a user.
+
+### Achievements
+
+To add some gamification aspects to the app to let the user keep going the app should have an individual score with virtual achievements. This means that every task that is complete will give the user a certain number of points. The baseline tasks should give more points because they are more important than the challenges and the points from challenges should be caped to a maximum per day. As a result, no one should be encouraged to prefer the challenges over the baseline task in respect to the points gained. 
```

Comment:
```suggestion
To add some gamification aspects to the app to let the user keep going the app should have an individual score with virtual achievements. For every completed task, the user is gaining a more points, based on the task. The baseline tasks should give more points because they are more important than the challenges and the points from challenges should be caped to a maximum per day. As a result, no one should be encouraged to prefer the challenges over the baseline task in respect to the points gained. 
```